extensions/wikibase/tests/src/org/openrefine/wikibase/commands/PreviewWikibaseSchemaCommandTest.java

```diff
@@ -110,4 +120,33 @@ public void testInvalidManifest() throws IOException, ServletException {
         assertEquals(writer.toString(),
                 "{\"code\":\"error\",\"message\":\"Wikibase manifest could not be parsed. Error message: invalid manifest format\"}");
     }
+
+    @Test
+    public void testWarningData() throws Exception {
```

Comment:
Very cool that you were able to merge this back into the existing test class!

Reply:
Thanks